### PR TITLE
Application code

### PR DESCRIPTION
**Pull request by ghost**
Reviewing my past work

### CODE REVIEW

**Comment by ghost** [comment]
Congratulations on getting your code approved! But some changes must still be made for the website to be completed:
1- No linters were used in the project
2- GitFlow was not organized
3- The page is overall hard to read

**Comment by ghost** [comment]
`<section id="introduction" class="main-section">`
Many elements have ids and classes, which is not necessary. One of them is enough.

**Comment by ghost** [comment]
```
<section id="what_you_should_already_know" class="main-section">
        <header id="what_you_should_already_know">
```
ids cannot be repeated, classes can. This must be corrected in order to be approved.

**Comment by ghost** [comment]
`<p>
          
        </p>`
There is an empty paragraph between lines 81 and 83, and another between 92 and 94, 103 and 105, and some others. Please correct this.

**Comment by ghost** [comment]
<h1>Hello World</h1>
There is an h1 tag in the end of the document, by the end of the main section

**Comment by ghost** [comment]
no css was applied to the document. A style.css was uploaded empty to the repo.

**Comment by ghost** [comment]
Your readme does not contain the name of the author(s), and is not very well documented.

**Review by AleksandarIvezic — CHANGES_REQUESTED** [review]
## Hi @iKuartz 👋🏼 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️♻️♻️
_Check the comments under the review._

- Please make sure that you communicate politely:
   - Instead of giving orders, make suggestions. "I think it would be better to use classes instead of ids, because..." sounds nicer than "ids cannot be repeated, classes can. This must be corrected in order to be approved." doesn't it? 
   - Remember to explain why you request a change - it might be obvious for you but not for someone else. This comment doesn't have any explanation: "There is an h1 tag in the end of the document, by the end of the main section."
- Please make sure that you know how to use the GitHub features:
  - You should use inline comments so the author of the project can easily understand the context of your comment. Watch this [video with an example](https://www.loom.com/share/e847c46aa4814534b23b3a8e4dd60d6d).
  
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- To include emojis and gifs to make your code review more human-friendly 👽.
- To use [REQUIRED] and  [OPTIONAL] prefixes to point out the importance of changes you propose.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Also, don't hesitate to contact me over slack.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: html_css_proj/index.html [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+          Introduction
+        </header>
+        <p>
+          <code>$(element).css({backgroundColor:'transparent'})</code>
+        </p>
+        <p>
+          
+        </p>
+      </section>      
+      <section id="what_you_should_already_know" class="main-section">
```
Comment (ghost) [inline html_css_proj/index.html:56]: [REQUIRED] Please, avoid using "id" and "class" on the same HTML tag. They work for the same purpose, but "class" can be repeated and used in other tags, while id cannot.

File: html_css_proj/index.html [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+        <header id="what_you_should_already_know">
+          What you should already know
+        </header>
+        <p>
+          <code>$(header).css({Color:'Red'})</code>
+        </p>
+        <p>
+          
+        </p>
```
Comment (ghost) [inline html_css_proj/index.html:65]: [REQUIRED] Please pay attention to empty paragraphs such as this, and kindly remove them from the code. If you need a line break, the semantically correct way to do it is by adding a <br> tag.

File: html_css_proj/index.html [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+        <p>
+          <code>$(header).css({Color:'Purple'})</code>
+        </p>
+        <p>
+          
+        </p>
+      </section>
+
+    </main>
+    <h1>Hello World</h1>
```
Comment (ghost) [inline html_css_proj/index.html:109]: [REQUIRED] This h1 tag looks out of place 🧐. It should define the most important text of a page, so how about positioning it higher. A suggestion would be to use it in the title.

**Review by ghost — COMMENTED** [review]
Good job so far! Just a few more changes to go!

**Review by lily-coder — APPROVED** [review]
Hi @iKuartz , 

Great job implementing the suggestions by the previous reviewer. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**